app/templates/consortium/content-column.html

```diff
@@ -1,34 +1,34 @@
 <!-- -------------------------------------------------------------------------------------------------------------- -->
 <h1 id="mission">Mission</h1>
 <p>
-  Being an open project, UAVCAN is critically dependent on direct contributions from its adopters.
-  This Consortium is a non-profit organization created to ensure the future of UAVCAN and to provide a
+  Being an open project, OpenCyphal is critically dependent on direct contributions from its adopters.
+  This Consortium is a non-profit organization created to ensure the future of OpenCyphal and to provide a
   neutral collaboration platform for the stakeholders.
   The core objectives of the Consortium are:
 </p>
-<img src="/static/images/uavcan-conformity-mark.svg" width="160" height="160" class="img-fluid float-right p-3"
-     alt="UAVCAN Conformity Mark"
-     title="UAVCAN Conformity Mark">
+<img src="/static/images/cyphal-conformity-mark.svg" width="160" height="160" class="img-fluid float-right p-3"
+     alt="Cyphal Conformity Mark"
```

Comment:
There is no Cyphal conformity mark right now. Please be careful with this term. The community logo cannot be a conformity mark.

Reply:
This part is commented out. Please use [uat.opencyphal.org:8080](http://uat.opencyphal.org:8080/) for review, not the source code.